bin/pytorch_inference/Main.cc

```diff
@@ -120,7 +120,7 @@ void writeError(const std::string& requestId,
                 const std::string& message,
                 ml::core::CRapidJsonConcurrentLineWriter& jsonWriter) {
     jsonWriter.StartObject();
-    jsonWriter.Key(RESULT);
+    jsonWriter.Key(ERROR);
```

Comment:
Now errors are written as errors not a special type of results with an error message. On the Java parsing side this change makes the code clearer.